.vsts-dotnet-ci.yml

```diff
@@ -86,7 +86,7 @@ jobs:
     pool:
       name: NetCore-Public
       demands: ImageOverride -equals build.ubuntu.1804.amd64.open
-    helixQueue: Ubuntu.1804.Amd64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:centos-7-mlnet-helix
+    helixQueue: Ubuntu.1804.Amd64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:centos-stream8-mlnet-helix
```

Comment:
Technically I think this can now be reverted.  We should still be able to run our tests that target .NET 6.0 on net6 on centos 7.

I previously needed to change this to stream8 because the 9.0 SDK was being deployed in helix, but that's fixed with https://github.com/dotnet/machinelearning/commit/b8f71b9c661b5dea98ce756a9784ee1626d5dd38.

@michaelgsharp what are your thoughts -- should I revert this change?

Reply:
I went ahead and undid this in order to test it.

Reply:
Ok, locally I'm seeing this one failure when using the centos7 image:
```
[31;1m    Microsoft.ML.RunTests.TestEntryPoints.EntryPointLightLdaTransformer [FAIL]
[m[37m      System.DllNotFoundException : Unable to load shared library 'LdaNative' or one of its dependencies. In order to help diagnose loading problems, consider setting the LD_DEBUG environment variable: libLdaNative: cannot open shared object file: No such file or directory
```

With LD_DEBUG enabled:
```
      2929:	file=/helix/workitems/Microsoft.ML.Core.Tests/libLdaNative.so [0];  dynamically loaded by /helix/correlation-payload/dotnet-cli/shared/Microsoft.NETCore.App/6.0.9/libcoreclr.so [0]
      2929:	file=/helix/workitems/Microsoft.ML.Core.Tests/libLdaNative.so [0];  generating link map
      2929:	  dynamic: 0x00007f6da47f7d80  base: 0x00007f6da45e0000   size: 0x0000000000218218
      2929:	    entry: 0x00007f6da45e2f40  phdr: 0x00007f6da45e0040  phnum:                  7
      2929:	
      2929:	checking for version `GCC_3.0' in file /lib64/libgcc_s.so.1 [0] required by file /helix/workitems/Microsoft.ML.Core.Tests/libLdaNative.so [0]
      2929:	checking for version `GLIBC_2.2.5' in file /lib64/libm.so.6 [0] required by file /helix/workitems/Microsoft.ML.Core.Tests/libLdaNative.so [0]
      2929:	checking for version `GLIBC_2.4' in file /lib64/libc.so.6 [0] required by file /helix/workitems/Microsoft.ML.Core.Tests/libLdaNative.so [0]
      2929:	checking for version `GLIBC_2.14' in file /lib64/libc.so.6 [0] required by file /helix/workitems/Microsoft.ML.Core.Tests/libLdaNative.so [0]
      2929:	checking for version `GLIBC_2.3.4' in file /lib64/libc.so.6 [0] required by file /helix/workitems/Microsoft.ML.Core.Tests/libLdaNative.so [0]
      2929:	checking for version `GLIBC_2.2.5' in file /lib64/libc.so.6 [0] required by file /helix/workitems/Microsoft.ML.Core.Tests/libLdaNative.so [0]
      2929:	checking for version `GLIBCXX_3.4.17' in file /lib64/libstdc++.so.6 [0] required by file /helix/workitems/Microsoft.ML.Core.Tests/libLdaNative.so [0]
      2929:	checking for version `GLIBCXX_3.4.9' in file /lib64/libstdc++.so.6 [0] required by file /helix/workitems/Microsoft.ML.Core.Tests/libLdaNative.so [0]
      2929:	checking for version `CXXABI_1.3.8' in file /lib64/libstdc++.so.6 [0] required by file /helix/workitems/Microsoft.ML.Core.Tests/libLdaNative.so [0]
      2929:	/lib64/libstdc++.so.6: error: version lookup error: version `CXXABI_1.3.8' not found (required by /helix/workitems/Microsoft.ML.Core.Tests/libLdaNative.so) (fatal)
```

Reply:
That's not the only failure - latest helix scripts no longer run on this platform either.  I reverted this.